src/Bicep.LangServer/Completions/ParamsCompletionContext.cs

```diff
@@ -73,14 +87,20 @@ public static ParamsCompletionContext Create(ParamsCompilationContext paramsComp
             // using fo|o
             SyntaxMatcher.IsTailMatch<UsingDeclarationSyntax, SkippedTriviaSyntax, Token>(matchingNodes, (_, _, token) => token.Type == TokenType.Identifier);
 
-        private static bool IsParamAssignmentContext(List<SyntaxBase> matchingNodes, int offset) =>
+        private static bool IsParamIdentifierContext(List<SyntaxBase> matchingNodes, int offset) =>
             // param |
             SyntaxMatcher.IsTailMatch<ParameterAssignmentSyntax>(matchingNodes, (paramAssignment  => paramAssignment.Name.IdentifierName == LanguageConstants.MissingName)) ||
             // param | = 
             SyntaxMatcher.IsTailMatch<ParameterAssignmentSyntax>(matchingNodes, (paramAssignment  => paramAssignment.Name.IdentifierName == LanguageConstants.ErrorName)) ||
             // param t|
             SyntaxMatcher.IsTailMatch<ParameterAssignmentSyntax, IdentifierSyntax, Token>(matchingNodes, (_, _, token) => token.Type == TokenType.Identifier);
 
+        private static bool IsParamValueContext(List<SyntaxBase> matchingNodes, int offset) =>
+            // param myParam = | 
+            SyntaxMatcher.IsTailMatch<ParameterAssignmentSyntax>(matchingNodes, (paramAssignment  => paramAssignment.Name.IdentifierName != LanguageConstants.MissingName || paramAssignment.Name.IdentifierName != LanguageConstants.ErrorName))||
+            // param myParam = 't|'
+            SyntaxMatcher.IsTailMatch<ParameterAssignmentSyntax, StringSyntax, Token>(matchingNodes, (paramAssignment, _, _)  => paramAssignment.Name.IdentifierName != LanguageConstants.MissingName || paramAssignment.Name.IdentifierName != LanguageConstants.ErrorName);
```

Comment:
This only matches for partially typed strings but I think that is fine since I am only supporting allowed value for string parameters right now?